tests/unit/package_managers/test_gomod.py

```diff
@@ -2215,8 +2215,7 @@ def test_retry(
         assert mock_sleep.call_count == tries_needed - 1
 
         for n in range(tries_needed - 1):
-            wait = 2**n
-            assert f"Backing off run_go(...) for {wait:.1f}s" in caplog.text
+            assert "Backing off _run(...) for " in caplog.text
```

Comment:
Like I mentioned in a different comment, this is very much appreciated in tests, but sadly won't make the **actual** cachi2 log output problem coming from Backoff go away.

Reply:
>  I would have dropped all string verifications because this functionality should be tested on backoff side, there is no good reason for re-testing someone else's code.

Agreed. "My first issue" material? :)